.editorconfig

```diff
@@ -3,7 +3,7 @@
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
All of our Python across all of our apps uses 4 spaces. If you wanna do this you should add a section for Python files.